src/components/BottomDrawer.tsx

```diff
@@ -25,13 +26,15 @@ interface BottomDrawerProps {
     name: string;
   }[];
   listItemClick: (event: React.MouseEvent, name: string) => void;
+  isDropout: boolean;
```

Comment:
@Rushikesh-Sonawane99 keep this component as generic component, dont mix any feature specific ui here

Reply:
@itsvick Created generic component and updated code